src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomColorIndex = new Random().nextInt(Colors.values().length);
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/b7d9d68c-ef40-4e1b-a7a2-a45947f62c8f)

Reply:
not fixed

Reply:
updated